op_crates/web/00_webidl.js

```diff
@@ -662,10 +662,30 @@
     };
   }
 
-  window.__bootstrap = window.__bootstrap || {};
+  // https://heycam.github.io/webidl/#es-enumeration
+  function createEnumConverter(name, ...values) {
+    const E = new Set(values);
+
+    return function (V, opts = {}) {
+      const S = String(V);
+
+      if (false === E.has(V)) {
```

Comment:
```suggestion
      if (!E.has(V)) {
```

Reply:
I'm surprised; imho, it's much more explicit as is, but I'll change.

Reply:
I read `if (!E.has(V)` as `if not E contains V`. The other one I would read as `if false is equal to E contains V`. First one seems simpler.

Reply:
Can you take another look now?

Reply:
I don't really like early returns for the `default` case. I feel you should early return for the invalid case, and then return the `default` case at the end of the function. I don't feel strongly enough about it to ask you to change it again, so lets keep it like it is now.